sample-apps/react-native/dogfood/ios/StreamReactNativeVideoSDKSample.xcodeproj/project.pbxproj

```diff
@@ -282,7 +281,6 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "set -e\nexport SENTRY_PROPERTIES=sentry.properties\nexport EXTRA_PACKAGER_ARGS=\"--sourcemap-output $DERIVED_FILE_DIR/main.jsbundle.map\"\nWITH_ENVIRONMENT=\"../node_modules/react-native/scripts/xcode/with-environment.sh\"\nREACT_NATIVE_XCODE=\"../node_modules/react-native/scripts/react-native-xcode.sh\"\nSENTRY_CLI_PATH=\"../node_modules/@sentry/cli/bin/sentry-cli\"\n/bin/sh -c \"$WITH_ENVIRONMENT \\\"$SENTRY_CLI_PATH react-native xcode $REACT_NATIVE_XCODE\\\"\"\n";
```

Comment:
Has this change been checked with what is in the default template..

This shell script uses `../node_modules/react-native/scripts/react-native-xcode.sh`

maybe something is there by default.. we have removed it to execute nothing here..

double check with the default template please

Reply:
It was added by you here - https://github.com/GetStream/stream-video-js/pull/109/files#diff-34aac08dc5adb010a4802828f350769160f6899cb0e60c3898ff87379c90db57R276

Reply:
After checking with [this](https://github.com/GetStream/stream-video-js/pull/109/files#diff-34aac08dc5adb010a4802828f350769160f6899cb0e60c3898ff87379c90db57R276) I found that I missed replacing it with the default template. I have pushed it now. Please check again. Thanks for pointing out. 👍🏻